.github/workflows/build.yml

```diff
@@ -35,3 +35,10 @@ jobs:
         with:
           name: 'backstop_test_results'
           path: '${{ github.workspace }}/backstop/results/'
+
+      - name: Host test results on gh pages
+        uses: peaceiris/actions-gh-pages@v3.7.3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ${{ github.workspace }}/backstop/results
+          destination_dir: ${{ github.event.number }}
```

Comment:
This is the pull request number.

So for #5, the action ran and hosted it at https://itwin.github.io/iTwinUI/5/html_report/

Reply:
I can look into to making it nicer tomorrow where you just have to click a link rather than manually go to the url.

Reply:
Added a Check called "Test results" where you can click the link to see report